WordPressAuthenticator/Authenticator/WordPressAuthenticator.swift

```diff
@@ -210,18 +239,19 @@ public struct WordPressAuthenticatorConfiguration {
 
     /// Designated Initializer
     ///
-    private init(configuration: WordPressAuthenticatorConfiguration) {
+    private init(configuration: WordPressAuthenticatorConfiguration, style: WordPressAuthenticatorStyle) {
         self.configuration = configuration
+        self.style = style
     }
 
     /// Initializes the WordPressAuthenticator with the specified Configuration.
     ///
-    public static func initialize(configuration: WordPressAuthenticatorConfiguration) {
+    public static func initialize(configuration: WordPressAuthenticatorConfiguration, style: WordPressAuthenticatorStyle) {
```

Comment:
Can we have `style: WordPressAuthenticatorStyle  = .defaultStyle` ?

Reply:
✔️